code/modules/vending/_vending.dm

```diff
@@ -766,6 +796,7 @@ IF YOU MODIFY THE PRODUCTS LIST OF A MACHINE, MAKE SURE TO UPDATE ITS RESUPPLY C
 	var/list/data = list()
 	data["onstation"] = onstation
 	data["department_bitflag"] = dept_req_for_free
+	//data["jobDiscount"] = 0.2
```

Comment:
What is this meant to to do and is there any other code associated with it? If it doesn't function at all, then it shouldn't really be included

Reply:
20% discount per department, in decimal form. 

Its not that its nonfunctional, rather that I did not want to add content to a fix pr. 

I will remove the commented code.